include/quicr/quicr_common.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#include <quicr_name>
+#include <qname>
```

Comment:
This may be a comment for the `qname` repo and not here, but I would really prefer if non-system headers had a suffix (preferably `.h`) to distinguish them from system headers.